extensions/amp-base-carousel/1.0/base-carousel.jss.js

```diff
@@ -54,9 +54,16 @@ export const arrowPlacement = {
   top: '50%',
   transform: 'translateY(-50%)',
   alignItems: 'center',
+  pointerEvents: 'auto',
+};
+const arrowLeft = {left: 0};
+const arrowRight = {right: 0};
```

Comment:
Let's stick with `arrowPrev` and `arrowNext` naming because the `left`/`right` placement relies on both slide direction (what's chronologically before/after) and document direction (ltr/rtl)

Reply:
There's not rtl support yet, but for this detail it's just be a transform on the arrows:
https://github.com/ampproject/amphtml/blob/57fa3f7a8dda48ce4b2488afa2c1a719190b0cfb/extensions/amp-base-carousel/0.1/amp-base-carousel.css#L56-L60